p2p/net/network.go

```diff
@@ -165,27 +165,38 @@ func (n *Net) publishClosingConnection(connection ConnectionWithErr) {
 	n.clsMutex.RUnlock()
 }
 
-func dial(ctx context.Context, address net.Addr) (net.Conn, error) {
+func (n *Net) dial(ctx context.Context, address net.Addr) (net.Conn, error) {
 	// connect via dialer so we can set tcp network params
 	dialer := &net.Dialer{}
 	netConn, err := dialer.DialContext(ctx, "tcp", address.String())
 	if err == nil {
 		tcpconn := netConn.(*net.TCPConn)
-		tcpSocketConfig(tcpconn)
+		errs := tcpSocketConfig(tcpconn)
```

Comment:
the errs list is a bit weird.
also, I would expect tcpConn to have a "config" or "init" method  instead of a configure method accepting a connection (if you do the first you would also be able to simply log these errors locally)

Reply:
TCP conn is a struct from the std lib. I can't add methods to it :\